src/Database/services/user.js

```diff
@@ -0,0 +1,8 @@
+import Models from '../../Database/models/server';
+
+const { User } = Models
+
+export default async (param) => {
```

Comment:
Every service must be a class such as `UserServices` and the function can be the method such as `findOne`